packages/base/src/BusyIndicatorEnablement.js

```diff
@@ -0,0 +1,53 @@
+import merge from "./thirdparty/merge.js";
+import {
+	isTabPrevious,
+} from "./Keys.js";
+
+const busyIndicatorMetadata = {
+	properties: {
+		__isBusy: {
+			type: Boolean,
+		},
+	},
+};
+
+const enrichMetadata = UI5Element => {
+	UI5Element.metadata = merge(UI5Element.metadata, busyIndicatorMetadata);
+};
+
+const enrichMethods = UI5ElementPrototype => {
+	Object.defineProperties(UI5ElementPrototype, {
+		"__redirectFocus": { value: true, writable: true },
+		"__suppressFocusBack": {
+			get: function () { // eslint-disable-line
```

Comment:
remove the eslint disable comment, use an arrow function, remove the `bind(this)` part